pkgs/shells/fish/plugins/fifc.nix

```
@@ -0,0 +1,23 @@
+{
+  lib,
+  buildFishPlugin,
+  fetchFromGitHub,
+}:
+buildFishPlugin rec {
+  pname = "fifc";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gazorby";
+    repo = "fifc";
+    rev = "v${version}";
+    sha256 = "sha256-p5E4Mx6j8hcM1bDbeftikyhfHxQ+qPDanuM1wNqGm6E=";
```

```suggestion
    hash = "sha256-p5E4Mx6j8hcM1bDbeftikyhfHxQ+qPDanuM1wNqGm6E=";
```
Please use `hash` instead of `sha256`.
https://nixos.org/manual/nixpkgs/stable/#fetchurl